src/component/mxgraph/BpmnMxGraph.ts

```diff
@@ -27,7 +27,7 @@ export class BpmnMxGraph extends mxgraph.mxGraph {
   /**
    * @internal
    */
-  constructor(readonly container: HTMLElement) {
+  constructor(container: HTMLElement) {
```

Comment:
The `container` is defined in the super class, putting `readonly` duplicated the definition in the subclass. `noImplicitOverride` makes TS generate an error in this case.
So I decided to no redefine the field here and make use of the one from the super class.